.github/workflows/deploy.yaml

```diff
@@ -189,7 +233,7 @@ jobs:
   build_hoprd_docker:
     name: Build HOPRD Docker
     runs-on: ubuntu-latest
-    needs: [build]
+    needs: [build, deploy_smart_contracts, publish_npm]
```

Comment:
do we await somewhere for the npm registry to update with the latest hoprd version?

Reply:
yes, the last step in the `publish_npm` job does that